packages/mc-scripts/src/commands/serve.ts

```diff
@@ -1,11 +1,27 @@
 import http from 'http';
 import serveHandler from 'serve-handler';
+import { processConfig } from '@commercetools-frontend/application-config';
 import paths from '../config/paths';
 
 const port = 3001;
 
 async function run() {
+  const applicationConfig = await processConfig();
+
   const server = http.createServer((request, response) => {
+    // When developing against local APIs, handle the login|logout routes separately.
+    if (applicationConfig.env.mcApiUrl.startsWith('http://localhost')) {
+      if (request.url?.startsWith('/login/authorize')) {
+        const redirectTo = new URL(request.url, applicationConfig.env.mcApiUrl);
+        response.writeHead(301, { Location: redirectTo.toString() }).end();
+        return;
+      }
+      if (request.url?.startsWith('/logout')) {
+        response.end('Please clear your session storage.');
```

Comment:
To keep it simple, if someone wants to "log out" they simply need to clear the session storage. Again, given that this is only needed internally when developing MC services we don't need to overcomplicate the implementation.